### DISCUSSION TITLE
Standalone config is not picked up like it should be

### DISCUSSION THREAD

**Discussion by jimmyd-be** [vercel/next.js#56999] [Help]
### Summary

I'm migrating our app to use docker. Therefore I try to configure standalone output in next.config.js but when I to next info it looks like that the output is not set to standalone but to N/A.
I use this config in next.config.js

module.exports = {
    output: 'standalone',
}

**Comment by icyJoseph** [comment]
Are you copying next.config.js into the builder environment?

**Reply by jimmyd-be** [reply]
When I do this on my Dev environment just running next build it does nothing.

**Reply by jimmyd-be** [reply]
Next.config.js is in the root of my project

**Reply by jimmyd-be** [reply]
Found it. I need to use the next_private_stabdalone en variable and this builds the standalone version
